test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -485,7 +485,9 @@ UCS_TEST_P(test_ucp_wireup_1sided, stress_connect) {
 }
 
 UCS_TEST_P(test_ucp_wireup_1sided, stress_connect2) {
-    int count = ucs_min(1000 / ucs::test_time_multiplier(), max_connections() / 2);
+    int count = ucs_min(1000 / (ucs::test_time_multiplier() *
```

Comment:
need to also set some minal count, let's say 10

Reply:
done